src/config/configurator.ts

```diff
@@ -0,0 +1,51 @@
+import { readFileSync } from "fs";
+import * as cdk from "aws-cdk-lib";
+import * as constructs from "constructs";
+
+class ConfiguratorAspect implements cdk.IAspect {
+  private readonly resourceType: string;
+  private readonly propertyName: string;
+  private readonly propertyValue: any;
+
+  constructor(resourceType: string, propertyName: string, propertyValue: any) {
+    this.resourceType = resourceType;
+    this.propertyName = propertyName;
+    this.propertyValue = propertyValue;
+  }
+  public visit(node: constructs.IConstruct): void {
+    if (cdk.CfnResource.isCfnResource(node) && node.cfnResourceType == this.resourceType) {
+      node.addPropertyOverride(this.propertyName, this.propertyValue);
+    }
+  }
+}
+
+export class Configurator {
+  private readonly config: any;
+  constructor(scope: constructs.Construct, config: string | object, environmentId?: string) {
```

Comment:
Is the idea to different config strategies (e.g. JSON, Yaml...) in a future PR?

Reply:
Correct

Reply:
Actually, will update to support YAML in this PR.